clients/da-vinci-client/src/main/java/com/linkedin/davinci/notifier/MetaSystemStoreReplicaStatusNotifier.java

```diff
@@ -39,7 +40,21 @@ private void report(String kafkaTopic, int partitionId, ExecutionStatus status)
       // No replica status reporting for meta system stores
       return;
     }
-    Store store = storeRepository.getStoreOrThrow(storeName);
+    Store store;
+    try {
+      store = storeRepository.getStoreOrThrow(storeName);
+    } catch (VeniceNoStoreException e) {
+      /**
+       * For store deletion, store removal in store repo might happen faster then {@link ExecutionStatus.DROPPED} action.
+       * This is to make sure the meta store notifier does not throw exception when this happens. For other status, this
+       * is not expected, so we should throw exception.
+       */
+      if (status.equals(ExecutionStatus.DROPPED)) {
+        LOGGER.warn("Store {} does not exist in store repository. Skip reporting status: {}", storeName, status);
```

Comment:
It seems like this is a benign scenario. If this is the right understanding, then I would prefer a lower log level than WARN. Perhaps INFO, or even DEBUG?

Reply:
Sure, will update it to INFO then. I don't think DEBUG will ever be printed out with current Venice Component logging level, since this is not excessive, I think INFO should be fine.